examples/basic_mysql_connection/index.js

```diff
@@ -13,8 +13,23 @@ var connection = mysql.createConnection({
 
 connection.connect();
 
-connection.query('SELECT * FROM todos', function (error, results, fields) {
-    console.log(results);
-});
+let query=[	'select count(*) as items from todos',
+			'select * from todos where done',
```

Comment:
I notice that you used `select *` instead of `SELECT *`. The convention is to use capital letters for sequel keywords - obviously the lowercase will still work, but the capital letters make it more readable, and readability counts 😀